src/Notes/BeforeCampaignMigration.php

```diff
@@ -0,0 +1,79 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Notes;
+
+use Automattic\WooCommerce\Admin\Notes\Note as NoteEntry;
+use Automattic\WooCommerce\GoogleListingsAndAds\Ads\AdsAwareTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\HelperTraits\Utilities;
+use Automattic\WooCommerce\GoogleListingsAndAds\Ads\AdsAwareInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\PluginHelper;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class BeforeCampaignMigration
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Notes
+ */
+class BeforeCampaignMigration extends AbstractNote implements AdsAwareInterface {
+
+	use AdsAwareTrait;
+	use PluginHelper;
+	use Utilities;
```

Comment:
💅 / ❓  

Are we really using `Utilities`? Or it is only for loading `OptionsAwareTrait`? If so, I guess we can just add `OptionsAwareTrait` instead of `Utilities`.

Reply:
Thanks for catching this! I have updated in [09c5a4f](https://github.com/woocommerce/google-listings-and-ads/pull/1508/commits/09c5a4fd03729ca2960c1214cf4a25a325779d47)